docs/src/terminology.md

```diff
@@ -24,6 +32,10 @@ A contiguous set of [entries](terminology.md#entry) on the ledger covered by a [
 
 A preimage resistant [hash](terminology.md#hash) of the [ledger](terminology.md#ledger) at a given [block height](terminology.md#block-height). Taken from the last [entry id](terminology.md#entry-id) in the slot
 
+## blockstore
+
+Before a block is considered to have reached finality, validators shall keep all valid forks that are generated through receipt of a shred signed by the expected leader for a given slot. Once a validator’s votes have reached max lockout, they may prune anything beyond the root from their blockstore. 
```

Comment:
The blockstore is a local database of recent blocks.

Reply:
I agree your definition is much more concise and clear, however doesn't seem to capture: https://docs.solana.com/cluster/overview#confirming-transactions

Reply:
Blockstore is lower level than the turbine block propagation strategy. When shreds are dropped by turbine and the block cannot be reassembled, the validator switches to its "repair" mechanism to retrieve the missing shreds. It gets those shreds from the blockstore held by other validators.